docs/collateral.md

```diff
@@ -109,96 +113,133 @@ interface ICollateral is IAsset {
+
+Note, this doesn't disqualify collateral with USD as its target unit! It's fine for the target unit to be the unit of account. This doesn't disqualify collateral with a non-USD target unit either! It's fine for the target unit to be different from the unit of account. These two concepts are totally orthogonal.
+
+### Representing Fractional Values
+
+Wherever contract variables have these units, it's understood that even though they're handled as `uint`s, they represent fractional values with 18 decimals. In particular, a `{tok}` value is a number of "whole tokens" with 18 decimals. So even though DAI has 18 decimals and USDC has 6 decimals, $1 in either token would be 1e18 when working in units of `{tok}`.
 
-Here all of these Target Baskets have been selected to start out roughly around \$1, assuming a slightly weak EURO and \$20k BTC. Over time however, each of these RTokens will have very different overall price trajectories from any vantage point.
+For more about our approach for handling decimal-fixed-point, see our [docs on the Fix Library](solidity-style.md#The-Fix-Library).
 
-Fortunately it's not so hard to figure out the target unit for a particular collateral plugin, at least once you've figured out the correct reference unit. **The target unit should be chosen such that the reference unit can be expected to roughly track the target unit, modulo short-term price deviations.** For USDC the target unit would be USD; for EURT it would be the EURO; for WBTC it would be BTC.
+## Synthetic Unit Example
 
-It is important to implement checks in the collateral plugin that the target unit is indeed tracking the reference unit. More on this later.
+Some collateral positions require a synthetic reference unit. This can be tricky to reason through, so here we'll provide a few examples.
 
-#### Unit of Account `{UoA}`
+### Using Uniswap V2 LP Tokens
 
-The Unit of Account `{UoA}` for a collateral plugin is entirely general _in principle_, but in practice it needs to be something that all the other collateral plugins used within an RToken can also support. Maybe in some future world we'll have enough EURO or big mac price information on-chain, but for now USD prices are king so it is probably correct to assume `UoA = USD`.
+Consider the Uniswap V2 LP token, **UNIV2LP**, for the USDC/USDT pair. (The following discussion assumes that you, reader, are familiar with the basic design of Uniswap V2. Their [documentation][univ2] is an excellent refresher.) Such a Collateral position might aim to earn revenue from liquidity fees, while maintaining a fully redeemable position in the two underlying fiatcoins.
```

Comment:
Turns out the token symbols are `UNI-V2`, not `UNIV2LP`. No mention of the tokens in the pair, though.